privacyidea/static/components/login/controllers/loginControllers.js

```diff
@@ -97,6 +97,22 @@ angular.module("privacyideaApp")
     $scope.piCustomBaselineFile = angular.element(document.querySelector('#CUSTOM_BASELINE')).val();
     // TODO: We can change this after login, if there is a user dependent customization!
 
+    $scope.confirmDelete = function(delete_function, identifier) {
+        $scope.confirmDeleteObj = {
+            question: document.activeElement.ariaLabel,
```

Comment:
The property `activeElement.ariaLabel` is not available in Firefox (see https://developer.mozilla.org/en-US/docs/Web/API/Element/ariaLabel#browser_compatibility).
Please use `activeElement.getAttribute('aria-label')`

Reply:
Done in f85deca5